models/src/Enemies.cpp

```diff
@@ -1,30 +1,67 @@
 #include "../include/Enemies.hpp"
-
+#include <RNG.hpp>
+const int SAFE_ZONE = 2;
+const int TOTAL_ENEMIES = 3; 
 Enemies::Enemies()
 {
-	populate(5);
+	// populate();
 }
 
 Enemies::~Enemies()
 {
 	for(auto &e:this->list)
 		delete e;
 }
-void Enemies::updateAll(float deltaTime, const Map &map)
+void Enemies::updateAll(float deltaTime, const Map &map, const Player &player)
 {
 	for(auto &e:this->list)
```

Comment:
Not sure why this is the case, must have been a previous merge, but this line has bad spacing.